connmgr/connmanager.go

```diff
@@ -526,9 +526,11 @@ func (cm *ConnManager) Start() {
 		}
 	}
 
-	for i := atomic.LoadUint64(&cm.connReqCount); i < uint64(cm.cfg.TargetOutbound); i++ {
-		go cm.NewConnReq()
-	}
+	go func() {
```

Comment:
Not sure what this is meant to accomplish. This is just wrapping `cm.NewConnReq()` in a go routine that executes immediately. I don't see anything that would prevent duplicate connections. I was expecting a mutex or some other lock to make them go serially.